### ISSUE TITLE
Error on traced file on 0.8.3.1

### ISSUE THREAD

**Issue by luke-han** [google/mediapipe#1788]
Hi. 
When I download the traced file from the iOS app built with the profile setting and upload it to viz.mediapipe.dev, the following error occurs on v0.8.3.1 and no problem on v0.8.2.

```
Loader Could not load trace 'mediapipe_trace_0.binarypb'.
Uploader Could not load trace 'mediapipe_trace_0.binarypb': Cannot read property 'name' of undefined
```

I attached the traced file of version 0.8.3.1.
[mediapipe_trace_0.binarypb.zip](https://github.com/google/mediapipe/files/6189928/mediapipe_trace_0.binarypb.zip)

Thanks

**Comment by sgowroji** [comment]
Hi @luke-han, Could you please elaborate the above issue with complete details like which solution are you trying, what are your steps and when is the error occurred. Thanks !

**Comment by luke-han** [comment]
@sgowroji 

Hi, I used my own custom graph, but I could reproduce with FaceEffectApp.

I added the following codes at the top of `mediapipe/graphs/face_effect/face_effect_gpu.pbtxt` and run FaceEffectApp on iOS 14.4.1 (iPhone 12 Pro Max)
```
profiler_config {
  trace_enabled: true
  enable_profiler: true
  trace_log_interval_count: 200
}
```

And, I downloaded the container and upload `AppData/Documents/mediapipe_trace_0.binarypb` file to the viz.mediapipe.dev.

Thanks!

**Comment by sgowroji** [comment]
It's working fine while running on IOS device. But facing issue on  viz.mediapipe.dev. Right?

**Comment by luke-han** [comment]
> It's working fine while running on IOS device. But facing issue on viz.mediapipe.dev. Right?

Yes that’s correct. I think some keys of the traced file are missing.

**Comment by fahadhaq** [comment]
Is there a solution for this? I am getting the same error currently. 
thanks in advance

**Comment by johnboiles** [comment]
Confirmed same issue on 0.8.3.2.

**Comment by txf-** [comment]
I can confirm this traces collected on Linux also present this problem

**Comment by samlaf** [comment]
Would also like to see this fixed.

**Comment by craterone** [comment]
I confirmed the same issue on macOS(0.8.3.2).

**Comment by zzmicer** [comment]
Is there any other method to see visualization of the trace.binarypb?

**Comment by diegoaguilartaio** [comment]
It is happening on macOS with hands and object detector.  Any updates on this?

**Comment by kuaashish** [comment]
Hi @luke-han, This issue is fixed in the latest release of mediapipe `8.11`, Could you please let us know if this is still an issue at other end. Thank you!
